runbotics/runbotics-orchestrator-ui/src/src-landing/components/Navbar/Navbar.tsx

```diff
@@ -31,7 +31,11 @@ const Navbar: FC<NavbarProps> = ({
 
     return (
         <nav className={styles.nav}>
-            <ul className={`${styles.navLinkWrapper} ${isMobileVisible ? styles.minimized : navMobileStyle}`}>
+            <ul
+                className={`${styles.navLinkWrapper} ${
+                    isMobileVisible ? styles.minimized : navMobileStyle
```

Comment:
navMobileStyle is a class or a stylesheet?

Reply:
It's conditional expression
![image](https://github.com/runbotics/runbotics/assets/44839807/1c75f897-365c-4154-996d-73dd6fe7f6f3)